ksqldb-rest-app/src/main/java/io/confluent/ksql/api/auth/JaasPrincipal.java

```diff
@@ -17,11 +17,14 @@
 
 import java.security.Principal;
 
-class ApiPrincipal implements Principal {
+/**
+ * Principal implementation created when authenticating with the JaasAuthProvider
+ */
+class JaasPrincipal implements Principal {
 
   private final String name;
 
-  ApiPrincipal(final String name) {
+  JaasPrincipal(final String name) {
```

Comment:
nit: validate params that will be stored in object state; ensuring object does not get into an invalid state.

Reply:
ack